src/components/Form/Form.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+class Form extends React.Component {
+
+  submitFn = ( event ) => {
+    event.preventDefault();
+
+    event.target.reset();  
+  }
+
+  render(){
+    return (
+      <form autoComplete="off" onSubmit={this.submitFn}>
+        <div className="row">
+          <div className="col">
+            <input type="text" className="form-control" />
```

Comment:
sortuj propsy zawse alfabetycznie, popraw to wszędzie
`<input className="form-control" / type="text" >`